modules/outbound-command-event-handler/src/application/handler.ts

```diff
@@ -38,11 +39,10 @@ import {
     ProcessSDKOutboundBulkPartyInfoRequestMessage,
     ProcessPartyInfoCallbackMessage,
     ProcessSDKOutboundBulkPartyInfoRequestCompleteMessage,
+    IBulkTransactionEntityRepo,
+    ICommandEventHandlerOptions,
```

Comment:
Why was `ICommandEventHandlerOptions` moved?

Reply:
I forgot to move it back along with the aggregate code, sorry . Fixed.